src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -100,6 +100,14 @@ def collect_tool_functions_in_module(m):
     return tools
 
 
+def collect_flow_entry_in_module(m, entry):
+    entry = entry.split(":")[-1]
+    func = getattr(m, entry, None)
+    if isinstance(func, types.FunctionType):
+        return func
+    return None
```

Comment:
Maybe directly raise user error to avoid failed afterwords in wired place?

Reply:
Refined.